src/escrow/escrow.service.ts

```diff
@@ -104,7 +104,7 @@ export class EscrowService {
         const walletSeed = cryptoRandomString({ length: 32, characters: this.chars });
         // Agent id needs to be lowercase for k8s pod rules
         const agentId = cryptoRandomString({ length: 22, characters: this.chars }).toLowerCase();
-        const agentApiKey = cryptoRandomString({ length: 32, characters: this.chars }); // TODO update DB to save apiKeys
+        // const agentApiKey = cryptoRandomString({ length: 32, characters: this.chars }); // TODO update DB to save apiKeys
```

Comment:
should we just delete this line?

Reply:
Happy to delete it. I only left it in because of the TODO.